.babelrc

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
### Your Readme
![readme](https://user-images.githubusercontent.com/93039359/188752222-b8869e66-33d4-4a5f-b58d-ad3632bab7bd.png)

- [ ] Please add all the necessary information in your readme file because you are using readme template but you didn't add any information about your project and about yourself. This readme file is the

Reply:
resolved